python/pyngraph/axis_set.cpp

```diff
@@ -27,5 +31,19 @@ void regclass_pyngraph_AxisSet(py::module m)
     axis_set.doc() = "ngraph.impl.AxisSet wraps ngraph::AxisSet";
     axis_set.def(py::init<const std::initializer_list<size_t>&>());
     axis_set.def(py::init<const std::set<size_t>&>());
+    axis_set.def(py::init<const std::vector<size_t>&>());
     axis_set.def(py::init<const ngraph::AxisSet&>());
+
+    axis_set.def("__len__", [](const ngraph::AxisSet& v) { return v.size(); });
+
+    axis_set.def("__iter__",
+              [](ngraph::AxisSet& v) { return py::make_iterator(v.begin(), v.end()); },
+              py::keep_alive<0, 1>()); /* Keep set alive while iterator is used */
+
+    axis_set.def("__str__", [](const ngraph::AxisSet& self) -> std::string {
```

Comment:
Let's turn this into a proper `__repr__`

Reply:
Done.